nuitka/build/include/nuitka/exceptions.h

```diff
@@ -1272,6 +1272,43 @@ NUITKA_MAY_BE_UNUSED static inline int EXCEPTION_MATCH_BOOL(PyThreadState *tstat
     }
 }
 
+NUITKA_MAY_BE_UNUSED static inline int EXCEPTION_GROUP_MATCH_BOOL(PyThreadState *tstate, PyObject *exceptions_got,
+                                                                  PyObject *exceptions_checked) {
+    CHECK_OBJECT(exceptions_got);
+    CHECK_OBJECT(exceptions_checked);
+
+#if PYTHON_VERSION >= 0x3b0
```

Comment:
The define should apply to the whole function. I already added a file `exception_groups.h`  that has that define itself, but is also included like this:

```
#if PYTHON_VERSION >= 0x3b0
#include "nuitka/exception_groups.h"
#endif
```

So move it to there, and maybe build off the code I already have in that file.